projects/RabbitMQ.Client/client/impl/SocketFrameHandler.cs

```diff
@@ -238,45 +238,23 @@ public void Close()
             }
         }
 
-        public InboundFrame ReadFrame()
+        public ValueTask<InboundFrame> ReadFrameAsync()
```

Comment:
Async suffix or not?

Reply:
https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/1296